silo/models.py

```diff
@@ -250,6 +275,7 @@ class ReadTypeAdmin(admin.ModelAdmin):
     list_display = ('read_type','description','create_date','edit_date')
     display = 'Read Type'
 
+from django.contrib.contenttypes.fields import GenericRelation
```

Comment:
This would have gone better at the top of the file.

Reply:
Oh yeah, right. I forgot to move that.

Reply:
Yes. Imports should be placed at the top.